kolibri/core/assets/src/styles/main.styl

```diff
@@ -165,3 +165,7 @@ button
   height: 100% !important
   background-color: black
   // @stylint on
+
+.ui-icon-button, .ui-icon
+  svg
+    vertical-align: middle
```

Comment:
Rather than a global style, could possibly make new `k-icon-button` and `k-icon` components that have a scoped deep selector?

Reply:
I could, but I could just add these 3 lines... :sob: